udata_front/theme/gouvfr/datagouv-components/src/composables/preview/useTabularApiProfile.ts

```diff
@@ -0,0 +1,50 @@
+import { MaybeRefOrGetter, ref, toValue, watchEffect } from "vue";
+import { getProfile } from "../../api/tabularApi";
+import type { Resource } from "../../types/resources";
+
+export type ColumnInfo = {
+  score: number;
+  format: string;
+  python_type: string;
+};
+
+export default function useProfileTabularApi(resource: MaybeRefOrGetter<Resource>) {
```

Comment:
Maybe use the same name as the file?
```suggestion
export default function useTabularApiProfile(resource: MaybeRefOrGetter<Resource>) {
```